packages/core/upload/admin/src/pages/App/components/Header.js

```diff
@@ -46,6 +46,7 @@ export const Header = ({
               defaultMessage: 'Folders navigation',
             })}
             breadcrumbs={breadcrumbs}
+            currentFolderId={folder?.id}
```

Comment:
Before you casted `id` into a `Number`: is this not necessary?

Reply:
I'm sorry I'm not sure I understand - could you be more specific please?

Reply:
<img width="569" alt="Screenshot 2022-07-28 at 18 18 38" src="https://user-images.githubusercontent.com/2244375/181588032-8f685400-8bd9-4f94-8fd9-f3998e7d62b3.png">

Before you were passing around `folder.id` as a `number`, but now as `string` and I was wondering if that makes a difference?

Reply:
I just checked, `folder.id` returns the `id` as a number 🤔